handlers/mcping/mcping.go

```diff
@@ -0,0 +1,26 @@
+package mcping
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"github.com/lordralex/absol/logger"
+	"os/exec"
+)
+
+func RunCommand(ds *discordgo.Session, mc *discordgo.MessageCreate, c *discordgo.Channel, cmd string, args []string) {
+	if len(args) == 0 {
+		return
+	}
+
+	server := args[0]
+
+	pythonCmd := exec.Command("python", "mcping.py", server)
+	out, err := pythonCmd.Output()
```

Comment:
Should use CombinedOutput since this doesn't capture stderr

Reply:
The script doesnt seem to use stderr, should I still use that?